apps/broomva/app/globals.css

```
@@ -1,240 +1,293 @@
-    --ag-bg-elevated: #232340;
-    --ag-bg-hover: #2a2a4a;
-    --ag-text-primary: #ffffff;
-    --ag-text-secondary: #a0a0b8;
-    --ag-text-muted: #6b6b80;
-    --ag-text-disabled: #4a4a5c;
+    --bv-blue: #5480c7;
+    --bv-blue-accent: #009bd8;
+    --bv-success: #00a24f;
+    --bv-warning: #dca81c;
+    --bv-danger: #d42325;
+    --bv-border-5: rgba(60, 66, 82, 0.07);
```

Stylelint (17.14.0)

[error] Expected "0.07" to be "7%" (alpha-value-notation)

(alpha-value-notation)